playbooks/robusta_playbooks/disk_pressure_enricher.py

```diff
@@ -0,0 +1,94 @@
+from robusta.api import *
+from robusta.integrations.resource_analysis.disk_analyzer import DiskTransformer
+
+
+@action
+def disk_pressure_enricher(event: NodeEvent):
+    """
+    Provides relevant disk information for troubleshooting disk pressure issues when the disk on a node is filling up.
+    """
+    node = event.get_node()
+    if not node:
+        logging.error(
+            f"cannot run disk pressure enricher on event with no node object: {event}"
+        )
+        return
+
+    blocks: List[BaseBlock] = []
+
+    # run disk-tools on node and parse its json output
+    pods_distribution_str_info = RobustaPod.run_privileged_pod(
+        node.metadata.name,
+        pod_image="us-central1-docker.pkg.dev/genuine-flight-317411/devel/disk-tools:1.0",
+        env=[
+            EnvVar(
+                name="CURRENT_POD_NAME",
+                valueFrom=EnvVarSource(fieldRef=ObjectFieldSelector(fieldPath="metadata.name"))
+            ),
+            EnvVar(
+                name="CURRENT_POD_NAMESPACE",
+                valueFrom=EnvVarSource(fieldRef=ObjectFieldSelector(fieldPath="metadata.namespace"))
+            )
+        ],
+        mount_host_root=True
+    )
+    pods_distribution_info = json.loads(pods_distribution_str_info)
+    pods_distribution = pods_distribution_info["pods_disk_distribution"]["pods_distribution"]
+
+    # report disk-tools warnings
+    for warning in pods_distribution_info["pods_disk_distribution"]["warnings"]:
```

Comment:
Perhaps best to report these as an enrichment too. Otherwise data can be inaccurate without any indication.

Reply:
Okay, I'm moving the warnings to the enrichment altogether- so there are no longer warning logs